apps/saleor-app-checkout/next.config.js

```diff
@@ -24,7 +27,7 @@ const config = withTM({
   },
   images: { domains: ["localhost"] },
   experimental: {
-    esmExternals: false,
+    esmExternals: "loose",
```

Comment:
It took me a moment to figure out what it does 😅 - can we add a link to the docs?
```suggestion
    // https://nextjs.org/docs/messages/import-esm-externals
    esmExternals: "loose",
```

Reply:
Should we add docs links for every setting in every config file for every tool we use? I don't see any point in doing it.

Reply:
yes, I get that - but this setting somehow is not documented in Next.js docs - for example, I've tried to search `esmExternals` and... **nothing** 🙈 

<img width="319" alt="CleanShot 2022-09-23 at 10 15 45@2x" src="https://user-images.githubusercontent.com/4144459/191919057-54afba0d-54bf-4781-a0bb-9e046561e299.png">